src/devices/src/legacy/rtc.rs

```diff
@@ -0,0 +1,32 @@
+use std::convert::TryInto;
+use vm_device::bus::MmioAddress;
+use vm_device::MutDeviceMmio;
+use vm_superio::{rtc_pl031::NoEvents, RTC};
+
+use utils::debug;
+
+pub struct RTCWrapper(pub RTC<NoEvents>);
+
+impl MutDeviceMmio for RTCWrapper {
+    fn mmio_read(&mut self, _base: MmioAddress, offset: u64, data: &mut [u8]) {
```

Comment:
Can we add tests for these, similar to the serial ones?

Reply:
Added some unit tests.